DigitalLearningSolutions.Data.Tests/Services/RecommendedLearningServiceTests.cs

```diff
@@ -187,9 +219,10 @@ public async Task GetRecommendedLearningForSelfAssessment_calls_learning_hub_api
                 .Returns(clientResponse);
 
             // When
-            var result =
-                (await recommendedLearningService.GetRecommendedLearningForSelfAssessment(SelfAssessmentId, DelegateId))
-                .ToList();
+            var _ = (await recommendedLearningService.GetRecommendedLearningForSelfAssessment(
+                SelfAssessmentId,
+                DelegateId
+            )).ToList();

```

Comment:
I'd assume we could just do             
`await recommendedLearningService.GetRecommendedLearningForSelfAssessment(SelfAssessmentId, DelegateId);`
 here